platform_versioned_docs/version-23.4/cli/commands.mdx

```diff
@@ -0,0 +1,1027 @@
+  FILE   | README_phase3_data_move_20150612           | 273      
+  FILE   | alignment.index                            | 3579471  
+  FILE   | analysis.sequence.index                    | 54743580 
+  FILE   | exome.alignment.index                      | 3549051  
+  FILE   | sequence.index                             | 67069489 
+  FOLDER | 1000G_2504_high_coverage/                  | 0        
+  FOLDER | alignment_indices/                         | 0        
+  FOLDER | changelog_details/                         | 0        
+  FOLDER | complete_genomics_indices/                 | 0        
+  FOLDER | data/                                      | 0        
+  FOLDER | hgsv_sv_discovery/                         | 0        
+  FOLDER | phase1/                                    | 0        
+  FOLDER | phase3/                                    | 0        
+  FOLDER | pilot_data/                                | 0        
+  FOLDER | release/                                   | 0        
+  FOLDER | sequence_indices/                          | 0        
+  FOLDER | technical/                                 | 0        
+  ```
+</details>
+<details>
+  <summary>**Organizations**</summary>
+
+  Run `tw organizations -h` to view supported workspace operations.
+
+  Organizations are the top-level structure and contain workspaces, members, and teams. You can also add external collaborators to an organization. See [Organization management](../orgs-and-teams/organizations.mdx) for more information. 
+
+  ### Add an organization
+
+  Run `tw organizations add -h` to view the required and optional fields for adding your workspace.
+
+  ```console 
+  tw organizations add -n TestOrg2 -f 2nd\ Test\ Organization\ LLC -l RSA
+
+  Organization 'TestOrg2' with ID '204336622618177' was added
+  ```
+
+</details>  
+<details>
+  <summary>**Members**</summary>
+
+  Run `tw members -h` to view supported member operations.
+
+  Manage organization members. 
+
+  ### List members 
+
+  Run `tw members list -h` view all the optional fields for listing organization members.
+
+  ```console
+  tw members list -o TestOrg2
+
+  Members for TestOrg2 organization:
+
+     ID              | Username             | Email                           | Role
+    -----------------+----------------------+---------------------------------+--------
+     200954501314303 | user1                | user1@domain.com                | MEMBER
+     277776534946151 | user2                | user2@domain.com                | MEMBER
+     243277166855716 | user3                | user3@domain.com                | OWNER
+
+  ``` 
+
+  ### Add a member 
+
+  Run `tw members add -h` view all the required and optional fields for adding organization members.
+
+  ```console
+  tw members add -u user1@domain.com -o DocTestOrg2
+
+  Member 'user1' with ID '134534064600266' was added in organization 'TestOrg2'
+  ```
+
+  ### Delete a member
```

Comment:
Does this require a level of permissions to execute this command (like `MAINTAIN` role and above)?